Source/WebCore/platform/network/curl/CurlMultipartHandle.cpp

```diff
@@ -79,12 +79,12 @@ CurlMultipartHandle::CurlMultipartHandle(CurlMultipartHandleClient& client, CStr
 {
 }
 
-void CurlMultipartHandle::didReceiveMessage(const SharedBuffer& buffer)
+void CurlMultipartHandle::didReceiveMessage(std::span<const uint8_t>&& receivedData)
```

Comment:
Grepping `std::span` through WebKit, you should remove `&&` here.

Reply:
Fixed